CHANGELOG.md

```diff
@@ -2,19 +2,16 @@
 
 ## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/0.15.1...HEAD)
 
-## [0.15.1](https://github.com/OpenLineage/OpenLineage/compare/0.15.0...0.15.1) - 2022-10-04
-### Fixed
-* CI: build macos release package on medium resource class [`#1131`](https://github.com/OpenLineage/OpenLineage/pull/1131) [@mobuchowski](https://github.com/mobuchowski)  
-    *Fixes failing build due to resource class being too large.*
-
-## [0.15.0](https://github.com/OpenLineage/OpenLineage/compare/0.14.1...0.15.0) - 2022-10-04
+## [0.15.1](https://github.com/OpenLineage/OpenLineage/compare/0.14.1...0.15.1) - 2022-10-05
 ### Added
 * Airflow: improve development experience [`#1101`](https://github.com/OpenLineage/OpenLineage/pull/1101) [@JDarDagran](https://github.com/JDarDagran)  
     *Adds an interactive development environment to the Airflow integration and improves integration testing.*
-* Documentation: update issue templates for proposal & add new integration template [`#1116`](https://github.com/OpenLineage/OpenLineage/pull/1116) [@rossturk](https://github.com/rossturk)  
-    *Adds a YAML issue template for new integrations and fixes a bug in the proposal template.*
 * Spark: add description for URL parameters in readme, change `overwriteName` to `appName` [`#1130`](https://github.com/OpenLineage/OpenLineage/pull/1130) [@tnazarew](https://github.com/tnazarew)  
```

Comment:
i think you can leave only 
* Spark: add description for URL parameters in readme[`#1130`](https://github.com/OpenLineage/OpenLineage/pull/1130) [@tnazarew](https://github.com/tnazarew)  
    *Adds more information about passing arguments with `spark.openlineage.url`*
    
I the stuff about renaming is for property that was created in this release so I don't think it should be mentioned here